### PR TITLE
docs(controller): add missing restful routes

### PR DESCRIPTION
**Pull request by RomainLanz**

### CODE REVIEW

File: 2.0/controllers.md [hunk 1]
```diff
@@ -42,7 +42,7 @@ Route.get('/users', 'UserController.index')
 
 ## Resourceful Controllers
 
-Resourceful routes define multiple routes and attach conventional methods to them under single route definition.
+[Resourceful routes](http://adonisjs.com/docs/2.0/routing#restful-routes) define multiple routes and attach conventional methods to them under single route definition.
```
Comment (thetutlage) [inline 2.0/controllers.md]: I cannot see any links to this

Comment (RomainLanz) [inline 2.0/controllers.md]: It's in the develop branch for the moment.

https://github.com/adonisjs/docs/blob/develop/2.0/routing.md#restful-routes

**Comment by thetutlage** [comment]
Resourceful routes are in controllers.md file, so this will not be required. I have moved your changes from `routes.md` to `controllers.md`.

**Comment by RomainLanz** [comment]
:+1:

**Pull request closed without merging**